src/main/drivers/pwm_output.h

```diff
@@ -29,6 +29,26 @@
 #endif
 
 typedef enum {
+    DSHOT_CMD_MOTOR_STOP = 0,
+    DSHOT_CMD_BEEP1,
+    DSHOT_CMD_BEEP2,
+    DSHOT_CMD_BEEP3,
+    DSHOT_CMD_BEEP4,
+    DSHOT_CMD_BEEP5,
+    DSHOT_CMD_ESC_INFO,
+    DSHOT_CMD_SPIN_ONE_WAY, 
+    DSHOT_CMD_SPIN_OTHER_WAY, 
+    DSHOT_CMD_3D_MODE_OFF,
+    DSHOT_CMD_3D_MODE_ON, 
+    DSHOT_CMD_SETTINGS_REQUEST,
+    DSHOT_CMD_SAVE_SETTINGS, 
+    DSHOT_CMD_ROTATE_NORMAL = 20, //Blheli_S only command
+    DSHOT_CMD_ROTATE_REVERSE = 21,  //Blheli_S only command
```

Comment:
Specifically the decoding of the ESC info: https://github.com/betaflight/betaflight/pull/3474/files#diff-72a6342f16b60f7944720f4dc08a71e1R2209
It seems to mostly match up when looking at what I get from ESCs, but the frame format specification I got is second hand, so not sure if that is complete and still accurate.